src/main/java/atmosfears/AtmosFearsBE/database/AirParticulatesFileRepository.java

```diff
@@ -0,0 +1,8 @@
+package atmosfears.AtmosFearsBE.database;
+
+import atmosfears.AtmosFearsBE.model.AirParticulatesFile;
+import org.springframework.data.mongodb.repository.MongoRepository;
+
+public interface AirParticulatesFileRepository extends MongoRepository<AirParticulatesFile, String> {
```

Comment:
```suggestion

@Repository
public interface AirParticulatesFileRepository extends MongoRepository<AirParticulatesFile, UUID> {
```

Add the `@Repository` decorator. I would also prefer to use `UUID` for ID type as it is designed for IDs.